src/main/java/ru/itis/services/ItemService.java

```diff
@@ -4,7 +4,7 @@
 import ru.itis.models.Item;
 
 public interface ItemService {
```

Comment:
Do we really need this interface?

Reply:
Is it better avoid using interfaces and simply use services implementations?

Reply:
I don't know, what made you to create it?

Reply:
Sidikov

Reply:
anyway fixed